### DISCUSSION TITLE
Russian translation for "Releases" in Traffic

### DISCUSSION THREAD

**Discussion by jpalus** [community/community#101815] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Noticed today that in one of GitHub projects Insights->Traffic shows "Releases" translated to Russian "Релизы". I'm not Russian, I don't speak Russian and have no clue where it came from or why only this specific item is mistranslated.

**Comment by DarrenChochian** [comment]


**Reply by jpalus** [reply]
Hopefully I'm not insulting anyone but very much looks like ChatGPT generated response :) Future is here I guess.

Happy to "report this bug to GitHub Support" but what does that mean exactly?

I don't think I ever changed any language preferences, they all seem default. Actually is there anything other than:

![image](https://github.com/community/community/assets/484155/279558b7-d00f-4e31-9009-ac2498683d4c)

**Comment by jpalus** [comment, marked as answer]
Looks like it got fixed recently. Thanks.
